db/migrate/20220920110226_add_user_ref_to_post.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddUserRefToPost < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
- [ ] Please follow the ERD provided for this tutorial, right now you are not affecting the Author as a reference for the user.